app/controllers/candidate_interface/section_controller.rb

```diff
@@ -1,8 +1,11 @@
 module CandidateInterface
   class SectionController < CandidateInterfaceController
+    before_action :redirect_to_post_offer_dashboard_if_accepted_or_recruited, if: :continuous_applications?
```

Comment:
I'm unable to use 'Request another reference' or 'cancel reference' – I'm redirected back to the dashboard with both.
<img width="613" alt="image" src="https://github.com/DFE-Digital/apply-for-teacher-training/assets/47917431/a494f60b-c3ad-4723-96c5-eb740ecfb651">

Reply:
Interesting, the tests didn't caught because all of them were continuous applications false. I know how to fix but I wish the tests caught that. I might need to see how can we make this specs continuous applications true.

Reply:
Pushed the changes (and adding specs as continuous applications as true!).